app/__main__.py

```diff
@@ -12,10 +14,33 @@
 
 dotenv.load_dotenv()
 TOKEN = os.environ["TOKEN"]
-PRIOTIY_COST: dict[MessagePriority, int] = {MessagePriority.BOTTOM: 500, MessagePriority.MIDDLE: 2500}
+PRIORITY_COST: dict[MessagePriority, int] = {
+    MessagePriority.BOTTOM: 500,
+    MessagePriority.MIDDLE: 2500,
+    MessagePriority.TOP: -1,
+}
+CPS_PIPELINE: list[float] = [x / 10.0 for x in range(1, 20001)]
```

Comment:
Do we really need this?

Reply:
Considering that each upgrade is going to be 0.1 more CPS, I think this is redundant and that we can just put it in the upgrade code